### PR TITLE
W-13480307 Changed bullets to number

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/delete-sla-tier-task.adoc [hunk 1]
```diff
@@ -4,13 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 
 . Create a new SLA tier:
-* In API Manager, in API Administration, click an API instance.
-* On the API dashboard, select SLA Tiers.
-* Set up and add the new SLA Tier.
+.. In API Manager, in API Administration, click an API instance.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/delete-sla-tier-task.adoc]:
```suggestion
.. In API Manager, in *API Administration*, click an API instance.
```

File: modules/ROOT/pages/defining-sla-tiers.adoc [hunk 2]
```diff
@@ -53,14 +54,30 @@ After you define an SLA for an API, you can approve or revoke access requests fr
 
 If you set the tiers for manual approval, email notifications are sent to you when developers request access for their applications. You can review the applications on the API version details page and approve, reject, or revoke requests. If a developer asks to change tiers, you can also review the change request and approve the application for the new tier or reject the change request.
 
-== Deprecating and Deleting Tiers
+== Deprecating a Tier
```
Comment (hannanelson) [inline modules/ROOT/pages/defining-sla-tiers.adoc]: For these H2 tasks, use imperative and singular (Deprecate a Tier).

File: modules/ROOT/pages/defining-sla-tiers.adoc [hunk 3]
```diff
@@ -53,14 +54,30 @@ After you define an SLA for an API, you can approve or revoke access requests fr
...
 
-After you assign applications to SLA tiers, you have the option to deprecate a tier to prevent developers from signing up new applications up for that tier. You cannot delete an SLA tier that has applications assigned to it.
+After you assign applications to SLA tiers, you have the option to deprecate a tier to prevent developers from signing up new applications for that tier. You cannot delete an SLA tier that has applications assigned to it.
 
-To deprecate and delete a tier, first create a new tier and notify application owners of the new tier name that you create. Next, revoke application access to the existing, deprecated tier. Ask the application owners to re-request access to the API using the new tier. Finally, after all application owners sign up for a new tier and you approve their move to the new tier, you can delete the deprecated tier.
+To deprecate a tier:
+
+. Create a new tier and notify application owners of the new tier name that you create.
```
Comment (hannanelson) [inline modules/ROOT/pages/defining-sla-tiers.adoc]:
```suggestion
. Create a new tier and notify application owners of the new tier name.
```

File: modules/ROOT/pages/defining-sla-tiers.adoc [hunk 4]
```diff
@@ -53,14 +54,30 @@ After you define an SLA for an API, you can approve or revoke access requests fr
...
+. Revoke application access to the existing, deprecated tier. 
+. Ask the application owners to re-request access to the API using the new tier.
+
+== Deleting a Tier
```
Comment (hannanelson) [inline modules/ROOT/pages/defining-sla-tiers.adoc]:
```suggestion
== Delete a Tier
```

File: modules/ROOT/pages/defining-sla-tiers.adoc [hunk 5]
```diff
@@ -53,14 +54,30 @@ After you define an SLA for an API, you can approve or revoke access requests fr
...
+After all application owners sign up for a new tier and you approve their move to the new tier, you can delete the deprecated tier.
+
+To delete a tier:
+
+. Navigate to *Anypoint Platform* > *API Manager*.
+. In the menu, select *API Administration*.
+. Click on an API Instance name.
```
Comment (hannanelson) [inline modules/ROOT/pages/defining-sla-tiers.adoc]: Should Instance be lowercase?

File: modules/ROOT/pages/defining-sla-tiers.adoc [hunk 6]
```diff
@@ -53,14 +54,30 @@ After you define an SLA for an API, you can approve or revoke access requests fr
...
+. On the API dashboard, select *SLA Tiers* from the menu.
+. For the tier you plan to delete, select *Delete* from the *More actions* menu.
```
Comment (hannanelson) [inline modules/ROOT/pages/defining-sla-tiers.adoc]:
```suggestion
. For the tier to delete, select *Delete* from the *More actions* menu.
```

**Review by hannanelson — DISMISSED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**